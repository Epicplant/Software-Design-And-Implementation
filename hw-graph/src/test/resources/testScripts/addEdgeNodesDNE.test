# Tests that adding an edge to nodes that don't exist won't break code

# Creates a new graph
CreateGraph newGraph

# Adds a new node
AddNode newGraph n1

# Adds an edge between two nonexistant nodes
AddEdge newGraph n1 n2 0

# Lists children of n1 which should be empty
ListChildren newGraph n1