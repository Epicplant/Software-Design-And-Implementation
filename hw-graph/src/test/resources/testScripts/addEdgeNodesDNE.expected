# Tests that adding an edge to nodes that don't exist won't break code

# Creates a new graph
created graph newGraph

# Adds a new node
added node n1 to newGraph

# Adds an edge between two nonexistant nodes
added edge 0 from n1 to n2 in newGraph

# Lists children of n1 which should be empty
the children of n1 in newGraph are: